internal/backend/usageupdater/dataposter.go

```diff
@@ -0,0 +1,46 @@
+package usageupdater
+
+import (
+	"bytes"
+	"crypto/tls"
+	"net/http"
+)
+
+type poster struct {
+	endpoint string
+	client   http.Client
+}
+
+func newPoster(endpoint string) poster {
+	tr := &http.Transport{
+		DisableKeepAlives:   true,
+		MaxIdleConnsPerHost: -1,
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
dummy implementation for now?

Reply:
nope, we just don't worry about it

Reply:
should we?

Reply:
not sure we care about the certificate here
I can remove the tis skip verify part
took reference from temporal code, they do the same

Reply:
ok